config/package/manifests/resources/eksclusterclass.resource.yaml

```diff
@@ -8,4 +8,4 @@ overview: |
 readme: |
  Cloud-specific resource classes are used to define a reusable configuration for a specific managed service. This class provides a managed Kubernetes cluster, which is satisfied by [Amazon Elastic Kubernetes Service (EKS)](https://aws.amazon.com/eks).
 
- Full documentation can be found in the Crossplane [API Reference docs](https://doc.crds.dev/github.com/crossplane/provider-aws/compute.aws.crossplane.io_eksclusterclasses.yaml).
\ No newline at end of file
+ Full documentation can be found in the Crossplane [API Reference docs](https://doc.crds.dev/github.com/crossplane/provider-aws/compute.aws.crossplane.io_eksclusterclasses.yaml@v0.10.0).
```

Comment:
@negz these links are actually wrong in `master` :/

They must match path of file in repo: https://github.com/crossplane/provider-aws/blob/5ee25e7076b99a4d1df39df9415f3071a667e8bc/config/stack/manifests/resources/s3bucketclass.resource.yaml#L11